internal/system/agent/main.go

```diff
@@ -66,6 +66,9 @@ func Main(ctx context.Context, cancel context.CancelFunc, router *mux.Router, re
 		configuration,
 		startupTimer,
 		dic,
+		// would need to change to true and add secretstore config in future once we have Consul access tightened up
+		// and thus required access token
+		false,
```

Comment:
I think this has to be `true` and add the SecretStore config section to the services config. Otherwise it will never be able to use Consul. This PR is all about enabling `all` the services to use secure consul.

Reply:
ok

Reply:
changed to true and added secretstore config